draft-ietf-rats-corim.md

```diff
@@ -159,7 +175,48 @@ The terminology from CBOR {{-cbor}}, CDDL {{-cddl}} and COSE {{-cose}} applies;
 in particular, CBOR diagnostic notation is defined in {{Section 8 of -cbor}}
 and {{Section G of -cddl}}. Terms and concepts are always referenced as proper nouns, i.e., with Capital Letters.
 
-{::boilerplate bcp14}
+This document uses the following terms:
+
+{: vspace="0"}
+Endorsed values:
+: A set of characteristics of an Attester that do not appear in Evidence.
+For example, Endorsed Values may include testing or certification data related to a hardware or firmware module.
+Endorsed Values are said to be "conditional" when they apply if other actual state matches Reference Values.
```

Comment:
I confirmed that this is indeed the definition of Conditional Endorsed Values. However, this is super confusing. I think what we really mean is something on the lines below:

` Endorsed values are said to be "conditional", when they apply if the claim from the Attester Target Environment (ie. its actual state) matches Reference Values`.

Reply:
Reference Values are corroborated and become accepted claims. Saying Reference Values gives the impression the condition applies to a Reference Value input.